contracts/Migrations.sol

```diff
@@ -15,9 +15,4 @@ contract Migrations {
   function setCompleted(uint completed) public restricted {
     last_completed_migration = completed;
   }
-
```

Comment:
This contract is auto-generated by Truffle and should be left as-is.

Reply:
OK, thanks for the guidance. I will recreat a new pull request later.